config/s3/config.go

```diff
@@ -32,7 +32,9 @@ func Configure(p *config.Provider) {
 	})
 
 	p.AddResourceConfigurator("aws_s3_bucket_acl", func(r *config.Resource) {
-		config.MoveToStatus(r.TerraformResource, "acl")
+		r.LateInitializer = config.LateInitializer{
+			IgnoredFields: []string{"acl", "access_control_policy"},
```

Comment:
Taking a note also in this PR to advocate for this. We can consider handling these conflicts automatically in upjet as the schema already has this information:
https://github.com/hashicorp/terraform-provider-aws/blob/baf25d816f9f2d5c463ab0d9bd267d28d15d1674/internal/service/s3/bucket_acl.go#L41
Will open a separate issue in upjet.

Reply:
@ulucinar that will be amazing, Alper, thanks!